aem-authoring-toolkit-samples/toolkit-samples-ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/samples/components/content/dungeons-component/dungeons-component.html

```diff
@@ -0,0 +1,15 @@
+<sly data-sly-use.data="com.exadel.aem.toolkit.samples.models.DungeonsComponent"></sly>
+<sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"></sly>
+<sly data-sly-call="${clientlib.all @ categories='authoring-toolkit.samples.components'}"></sly>
+
+<div class="dungeon-wrap">
+    <span class="dungeon-title">Dungeon</span>
+    <div class="dungeon-text-wrap">
+        <span class="dungeon-text">
+            <span>Once upon a time in the </span><span>${data.dungeonDescription @context='text'}</span>
```

Comment:
Do we need several `<span>`s side-by-side here? Will a space be visible between `..his adventure.` and `There are some..`?